Gopkg.toml

```diff
@@ -27,7 +27,7 @@
 
 [[constraint]]
   name = "github.com/blackducksoftware/hub-client-go"
-  source = "git@github.com:blackducksoftware/hub-client-go.git"
+#  source = "git@github.com:blackducksoftware/hub-client-go.git"
```

Comment:
Just remove it.  Don't bother commenting it out.  REmove the other source that is commented out too